### ISSUE TITLE
Docker image fails to run unless you add ":main" to the docker image name

### ISSUE THREAD

**Issue by emiltin** [rsmp-nordic/rsmp_validator#414]
The latest docker image ghcr.io/rsmp-nordic/rsmp_validator fails, but ghcr.io/rsmp-nordic/rsmp_validator:main works:

```
% docker run -it -v $PWD/config:/config -p 13111:13111 ghcr.io/rsmp-nordic/rsmp_validator
/bin/sh: 1: [: bundle,: unexpected operator

% docker run -it -v $PWD/config:/config -p 13111:13111 ghcr.io/rsmp-nordic/rsmp_validator:main
Run options: exclude {:sxl=>[unless relevant for 1.2.1], :core=>[unless relevant for 3.2.2]}
......................................................................................................

Finished in 14.81 seconds (files took 0.63707 seconds to load)
102 examples, 0 failures
```
 
It should be possible to use just ghcr.io/rsmp-nordic/rsmp_validator.

**Comment by emiltin** [comment]
I think it's related to how the metadata-action converts github events to docker tags in our docker build flow:

https://github.com/rsmp-nordic/rsmp_validator/blob/main/.github/workflows/docker.yaml#L37
https://github.com/docker/metadata-action

**Comment by emiltin** [comment]
Might be related to the default branch changing from 'master' to 'main. From the docs it seems the metadata-action still assumes the default branch is 'master'.

**Comment by emiltin** [comment]
Fixed with #415
